config/routes.rb

```diff
@@ -3,7 +3,7 @@
 require "sidekiq/web"
 
 Rails.application.routes.draw do
-  mount GraphiQL::Rails::Engine, at: "/graphiql", graphql_path: "/graphql" if Rails.env.development?
+  # mount GraphiQL::Rails::Engine, at: "/graphiql", graphql_path: "/graphql" if Rails.env.development?
```

Comment:
why 🤔 ?

Reply:
Sorry hahah, this pr is not done, I just marked ready for review to see if my commits will be counted. About this commented line it's because when I run the project, it raises a error about graphiQL, so I comment this line while I working on this issue. @edimossilva

Reply:
@philipeleandro , please create an issue explaining this bug you found

Reply:
@edimossilva Issue created #149